City-of-Helsinki/drupal-hdbt
# UHF-3424: Opening hours in unit teaser

# [UHF-3424](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3424)

## What was done

* Opening hours were re-introduced to the unit card
* Card layout was rewritten as the designs changed
* Card was renamed to content-liftup to follow the paragraph naming
* Card template was separated from tpr-unit template to its own file
* Supports swedish was removed from this card as requested by designer (confirmed with PO)
* Translations were added
* Extra spaces were removed from address template to reduce css complexity
* Layout logic was discussed and designed together with design, see comments below for approved screenshots

## How to test

* Get this branch
* Add content-liftup to landing page, basic page, in upper and lower regions, should work the same in all instances
  * Add unit with long opening hours
  * Add unit with short opening hours
  * Add unit without opening hours
  * Add unit with image
  * Add unit without image (sitewide fallback should be in use with custom color and icon)
* Check with 3 different breakpoint areas
  * 320px - 767px (mobile - breakpoint-m)
  * 768px - 991px (breakpoint-m - breakpoint-l)
  * 992px - (breakpoint-l - wide)
